src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -21,7 +21,9 @@ export const GithubCreateBranchOptionsGet = async (req: Request, res: Response,
 		return next(new Error(Errors.MISSING_ISSUE_KEY));
 	}
 
-	const jiraHost = getJiraHostFromTenantUrl(tenantUrl) || jiraHostQuery;
+	// TODO - once FF https://app.launchdarkly.com/jira/production/features/otc-arc-resolve-jwt-inplace-of-tenant_rdm2l/targeting
+	// has 100% roll out remove the fall back options and rely on res.locals.jirahost only.
+	const jiraHost = res.locals.jiraHost || getJiraHostFromTenantUrl(tenantUrl) || jiraHostQuery;
```

Comment:
Post flag 100% this will become.... and moved up
`const { jiraHost } = res.locals;`